subsys/debug/Kconfig

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
+config ARCH_HAS_GDBSTUB
```

Comment:
Wonder if this should be moved to `arch/Kconfig`? Most of `ARCH_HAS_*` are there.

Reply:
agreed

Reply:
yep, have done. Thanks !